cockroachcloud/console-access-management.md

```diff
@@ -32,11 +32,15 @@ Learn more about [managing SQL users' privileges](../{{site.versions["stable"]}}
 
 ## Roles
 
-Every {{ site.data.products.db }} user is either a Developer or a Console Admin for the organization.
+Every {{ site.data.products.db }} user is either a Developer or a Console Admin for the organization. 
+
+{{site.data.alerts.callout_danger}}
+Both Console Admins and Developers have access to all the information on the **SQL Activity** and **Databases** pages in the {{ site.data.products.db }} Console because they can create SQL users with the [`admin` role](../{{site.versions["stable"]}}/security-reference/authorization.html#admin-role).
```

Comment:
It's not because they can create SQL users with the admin role that they have access to the pages. It's because they are accessing the pages **using the admin role**. And the admin role has the highest privileges such as viewing all pages in the console _and_ creating SQL users. 

We can remove everything after Console...

Or if we want to be specific, change it to, "because they are part of the SQL admin role."

Reply:
gotcha, edited